### PR TITLE
work for 5.0.23

### PR DESCRIPTION
**Pull request by grahamegrieve**

### CODE REVIEW

File: org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/comparison/CapabilityStatementComparer.java [hunk 1]
```diff
@@ -0,0 +1,903 @@
...
+//      c = comp.getPropMap().get(c);
+//    }
+//    ConceptPropertyComponent cp = null;
+//    for (ConceptPropertyComponent t : cd.getProperty()) {
+//      if (t.getCode().equals(c)) {
+//        cp = t;
+//      }
+//    }
+//    return cp;
+//  }
```
Comment (markiantorno) [inline org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/comparison/CapabilityStatementComparer.java]: Do we need to leave the commented code in?

File: org.hl7.fhir.utilities/src/main/java/org/hl7/fhir/utilities/cache/PackageHacker.java [hunk 2]
```diff
@@ -116,15 +116,36 @@ private void addContentFrom(String folder, Map<String, byte[]> content) throws F
...
+    if ("file://C:\\GitHub\\hl7.fhir.us.odh#1.0.0\\output".equals(webref)) {                 return "http://hl7.org/fhir/us/odh/STU1"; }
+    if ("file://C:\\GitHub\\hl7.fhir.us.qicore#4.0.0\\output".equals(webref)) {              return "http://hl7.org/fhir/us/qicore"; }
+    if ("file://C:\\GitHub\\hl7.fhir.us.qicore#4.0.0\\output".equals(webref)) {              return "http://hl7.org/fhir/us/qicore/STU4"; }
+    if ("file://C:\\GitHub\\hl7.fhir.uv.ips#1.0.0\\output".equals(webref)) {                 return "http://hl7.org/fhir/uv/ips"; }
+    if ("file://C:\\GitHub\\hl7.fhir.uv.ips#1.0.0\\output".equals(webref)) {                 return "http://hl7.org/fhir/uv/ips/STU1"; }
+    if ("file://C:\\GitHub\\hl7.fhir.uv.mhealth-framework#0.1.0\\output".equals(webref)) {   return "http://hl7.org/fhir/uv/mhealth-framework/2020May"; }
+    if ("file://C:\\GitHub\\hl7.fhir.uv.security-label-ds4p#0.1.0\\output".equals(webref)) { return "http://hl7.org/fhir/uv/security-label-ds4p/2020May"; }
+    if ("file://C:\\GitHub\\hl7.fhir.uv.shorthand#0.12.0\\output".equals(webref)) {          return "http://hl7.org/fhir/uv/shorthand/2020May"; }
+
```
Comment (markiantorno) [inline org.hl7.fhir.utilities/src/main/java/org/hl7/fhir/utilities/cache/PackageHacker.java]: Might be better to use a switch statement here.

Comment (grahamegrieve) [reply org.hl7.fhir.utilities/src/main/java/org/hl7/fhir/utilities/cache/PackageHacker.java]: fixed. I'm not used to being able to do switch statements on strings

File: org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/comparison/CapabilityStatementComparer.java (hunk 1 above)
Comment (grahamegrieve) [reply org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/comparison/CapabilityStatementComparer.java]: oops removed. And there are unit tests

**Pull request merged**